XLS-30/README.md

```diff
@@ -116,13 +116,7 @@ Note that in this version, only equal-weighted two asset pools are supported. Ho
 
 ##### **`AccountRoot`** ledger entry
 
-A new flag **`lsfAMM`** is introduced to the [**`AccountRoot`**](https://xrpl.org/accountroot.html) object. 
-
-| Flag Name |  Hex Value | Description |
-| ---------- | :----------------: | :-------:|
-| `lsfAMM`      | 0x40000000  |   If set, it indicates that the object represents an AMM instance|
-
-Additionally, we introduce a new field called `AMMID` to the `AccountRoot` object. It replaces directory entry to link account root and AMM object. 
+We introduce a new field called `AMMID` to the `AccountRoot` object. It replaces directory entry to link account root and AMM object. 
```

Comment:
It doesn't replace the directory entry. We still have the directory entry. It does provide an easy link to AMM object and a way to identify AMM AccountRoot.

Reply:
Fixed